### PR TITLE
Fix the proto

### PR DESCRIPTION
**Pull request by asurkov**

### CODE REVIEW

File: prototype/narrator.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <!doctype html>
 <meta charset='utf-8'>
 <title>Narrator: an example of accessibility API consumer.</title>
-<script src="html.js"></script>
-<script src="aria.js"></script>
-<script src="a11yapi.js"></script>
-<script src="narrator.js"></script>
+<script type="application/javascript;version=1.8" src="html.js"></script>
```
Comment (marcoscaceres) [inline prototype/narrator.html:4]: Unfortunately, this confuses Chrome :(

File: prototype/a11yapi.js [hunk 2]
```diff
@@ -54,8 +56,9 @@
       }
 
       // Find an object whichever matches the DOM node best.
+      // Expected format: ':role(name) > tag[attr="value"]'
       let re =
-        /(?:\:role\((\w+)\))?(?:\s*>\s*)?(?:\*|(\w+))?(?:\[(\w+)(?:\='(\w+)')?\])?/;
+        /(?:\:role\((\w+)\))?(?:\s*>\s*)?(?:\*|(\w+))?(?:\[(\w+)(?:\=['"](\w+)['"])?\])?/;
```
Comment (marcoscaceres) [inline prototype/a11yapi.js:61]: This looks fine, I think.

**Comment by marcoscaceres** [comment]
I propose re-sending just the regex.

**Comment by marcoscaceres** [comment]
Adding this manually.

**Comment by asurkov** [comment]
ok. Today I've got closer in understading frond end developers :)

**Comment by marcoscaceres** [comment]
@asurkov I pushed the fixed regex directly onto gh-pages.

**Comment by asurkov** [comment]
Yeah, thank you. Any ideas how to reconcile Firefox with Chrome? :)

**Comment by marcoscaceres** [comment]
I complained in Bugzilla. I'll take a look if there is a way to enable
`let` by default that is not too painful in Gecko.

**Comment by marcoscaceres** [comment]
On June 10, 2015 at 8:56:44 PM, Alexander Surkov (notifications@github.com) wrote:

> ok. Today I've got closer in understading frond end developers :)

Heh, totally... I'm just in shock about the javascript-version thing! I never imagined it was that bad.

**Pull request closed without merging**